### ISSUE TITLE
[NuGet.org Bug]: Universal-FOSS-exception-1.0 license missing

### ISSUE THREAD

**Issue by fjssilva** [NuGet/NuGetGallery#10225]
### Impact

It's more difficult to complete my work

### Describe the bug

License identifier 'Universal-FOSS-exception-1.0' is an [approved SPDX license exception](https://spdx.org/licenses/exceptions-index.html) identifier but the corresponding license text cannot be found in [licences.nuget.org/Universal-FOSS-exception-1.0](https://licenses.nuget.org/Universal-FOSS-exception-1.0). Can you please add it?

### Repro Steps

Open the link https://licenses.nuget.org/Universal-FOSS-exception-1.0

### Expected Behavior

The link above is expected to show the contents of the Universal FOSS exception 1.0 license text, same as in https://spdx.org/licenses/Universal-FOSS-exception-1.0.html.

### Additional Context and logs

This is currently breaking the licencing information in [MySQL Connector/NET NuGet packages](https://www.nuget.org/packages/MySql.Data)

**Comment by erdembayar** [comment]
Thank you for filing this issue. Could you please add this license to here?
https://github.com/NuGet/NuGet.Licenses/tree/main/src/NuGet.Licenses/App_Data/Licenses

**Comment by fjssilva** [comment]
@erdembayar 
I just did in [PR #54](https://github.com/NuGet/NuGet.Licenses/pull/54), however I'm being asked to agree with Microsoft's CLA, which I can't without involving my company's Legal team and I'm afraid that may take a while. I have to ask, can you do it yourself? If you accept to do it, the fix is pretty simple: you just have to fix the script [src/NuGet.Licenses/license-text-extractor.ps1 like this](https://github.com/NuGet/NuGet.Licenses/pull/54/files#diff-829efa48fae8739d9bb925e14c8274081adee97b35b0c61e63b4b29d1fedc2fc) and run it.

**Comment by lyndaidaii** [comment]
@fjssilva, thank you for your feedback. We are currently working on this. Will get back to you once we get decision from legal team.

**Comment by fjssilva** [comment]
@lyndaidaii, the PR is finished and reviewed but stuck on the CLA issue. Can you please confirm if the PR can be accepted as-is? Or whether your are willing to reproduce the fix yourselves?
Thanks.

**Comment by lyndaidaii** [comment]
@agr is currently active looking into your PR.

**Comment by fjssilva** [comment]
Yes he is, and he's the one who approved the PR, however, his last comment was "@fjssilva, you'd need to https://github.com/NuGet/NuGet.Licenses/pull/54#issuecomment-2438160102 to CLA in order for this PR to be unblocked for merging."

My question is whether there's a way of skipping the CLA, for all the reasons I already explained. IIUC, two weeks ago you said something about checking with legal. Are there any news regarding this?

**Comment by erdembayar** [comment]
> Yes he is, and he's the one who approved the PR, however, his last comment was "@fjssilva, you'd need to [NuGet/NuGet.Licenses#54 (comment)](https://github.com/NuGet/NuGet.Licenses/pull/54#issuecomment-2438160102) to CLA in order for this PR to be unblocked for merging."
> 
> My question is whether there's a way of skipping the CLA, for all the reasons I already explained. IIUC, two weeks ago you said something about checking with legal. Are there any news regarding this?

@agr 's comment was about below CLA:
![image](https://github.com/user-attachments/assets/d8c94329-da70-42e9-8c0d-aa92a5d00f4a)

**Comment by agr** [comment]
The change is deployed to dev: https://licensesdev.nugettest.org/Universal-FOSS-exception-1.0
PROD deployment will likely happen after Thanksgiving.

**Comment by fjssilva** [comment]
> The change is deployed to dev: https://licensesdev.nugettest.org/Universal-FOSS-exception-1.0 PROD deployment will likely happen after Thanksgiving.

Any idea why this hasn't been deployed to production yet?

**Comment by agr** [comment]
Apologies, our deployment freeze started earlier and then this item fell of my radar. I'll start the process next week.

**Comment by fjssilva** [comment]
Great news! However, the way it's rendered on the page looks bad: 
![Image](https://github.com/user-attachments/assets/00bf2f96-3b91-430e-ac92-9c11d8b45a37).
You should be able to see this on https://www.nuget.org/packages/MySql.Data.

**Comment by agr** [comment]
That's a different issue tracked here: https://github.com/NuGet/NuGetGallery/issues/10341.
Deployments to all regions are complete. I'll close this one.
